doc/src/site/sphinx/deployment/scoring_mojo_pipeline.rst

```diff
@@ -3,8 +3,6 @@ Using the MOJO Scoring Pipeline with Spark/Sparkling Water
 
 MOJO scoring pipeline artifacts can be used in Spark to deploy predictions in parallel using the Sparkling Water API. This section shows how to load and run predictions on the MOJO scoring pipeline in Spark using Scala and the Python API.
 
-In the event that you upgrade H2O Driverless AI, we have a good news! Sparkling Water is backwards compatible with MOJO versions produced by older Driverless AI versions.
```

Comment:
Does it mean that we are not backwards compatible with older MOJO versions anymore?

Reply:
We are still backwards compatible, I just want to change place where we mention it